File: src/main/java/com/github/andreysum/lighting/controller/MainController.java
package com.github.andreysum.lighting.controller;

import javafx.fxml.FXML;
import javafx.scene.control.ComboBox;
import org.springframework.stereotype.Component;

/**
 * Handles saving category table controller.
 *
 * @author andreysum
 */
@Component
public class MainController {
    @FXML
    private ComboBox<String> input;

    @FXML
    public void initialize() {
        input.getItems().addAll("Any", "Second");
    }
}

File: src/main/kotlin/com/github/andreysum/lighting/controller/MainController.kt
package com.github.andreysum.lighting.controller

import com.github.andreysum.lighting.db.schema.TaskEntity
import com.github.andreysum.lighting.service.TaskListComboBoxInitializer
import javafx.fxml.FXML
import javafx.scene.control.ComboBox
import net.rgielen.fxweaver.core.FxmlView
import org.springframework.stereotype.Component


/**
 * Handles task input.
 *
 * @author andreysum
 */
@Component
@FxmlView("/javafx/main.fxml")
class MainController(
    private val initializer: TaskListComboBoxInitializer
) {
    @FXML
    lateinit var taskList: ComboBox<TaskEntity>

    @FXML
    fun initialize() {
        initializer.init(taskList)
    }
}